### DISCUSSION TITLE
Custom_lint_core limits analyzer to ">=5.7.0 <5.11.0"

### DISCUSSION THREAD

**Discussion by dngferreira** [rrousselGit/riverpod#2545] [Q&A]
Hey,
I was just checking a dependency problem on a project I'm building and traced it back to this dependency limitation on `custom_lint_core` that the `analyzer` version is between `5.7.0` and `5.11.0`.
Is there a reason for this limitation?
Cloning the repo locally it seems to work with the new `analyzer` version `5.12.0`.
But I'm new to dart and might be missing something.

**Comment by rrousselGit** [comment]
Should be good soon.
